docs/tctl/admin/db/clean.md

```diff
@@ -0,0 +1,113 @@
+---
+id: clean
+title: tctl admin db clean
+description: Cleaning up corrupted Workflows
+tags:
+  - reference
+  - tctl
+  - admin
+  - db
+---
+
+The `tctl admin db clean` command cleans corrupted [Workflow Executions](/docs/concepts/what-is-a-workflow-execution) from the targeted database.
+
```

Comment:
What do you think about having some sort of step by step instructions - general usage steps kind of thing?

Reply:
I definitely don't mind. If anything, I'd love to put instructions for all these pages, but in later updates to them